spec/controllers/activity_deletions_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe ActivityDeletionsController do
+  let(:activity) { create(:programme_activity, title: "", form_state: "purpose") }
+  let(:policy) { instance_double(ActivityPolicy) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(double("user"))
+    allow(ActivityPolicy).to receive(:new).and_return(policy)
+  end
+
+  describe "#new" do
+    context "when authorised" do
+      it "responds with a 200" do
+        allow(policy).to receive(:destroy?).and_return(true)
+
+        get :new, params: {organisation_id: activity.organisation, activity_id: activity.id}
+
+        expect(response.status).to eq(200)
```

Comment:
Are there some helpers that could be used here? 

`expect(response.status).to eq(:success)`

Reply:
Found them, they are `assert_response :success` ``assert_response :not_found` etc, but I don't think they are used anywhere in this project

Reply:
Another thing I did wonder about 😅 Are those helpers RSpec? I think you can do something like `have_http_status(:success)` or possibly just `eq(:success)`. The other controller specs have used status codes (which is why I went down this route), but we could update them